build/ci/cloudbuild.webdriver.yaml

```diff
@@ -64,18 +46,6 @@ steps:
       - |
         ./run_test.sh -w
 
-  # Run the Custom DC webdriver tests
```

Comment:
Since we're deleting this from the required webdriver check, we should strive to make the new cdc trigger required as quickly as possible.